### PR TITLE
#182201565 Setup docker for backend application 

### PR DESCRIPTION
**Pull request by Julienish14**
#### What does this PR do?
This pull request contains initial setup of docker for express server & postgress database 

#### How should this be manually tested?
First you have to install docker on your PC and then follow the steps to run the file: 
You have to add the ".env" file in the cloned RP and add the information form ".env.example" to be able to get access on postgress and port in docker compose file, 
To run docker compose write this code in your terminal " docker composer up " this will run the express server return the listening port message, and for postgress which is in progress it will show it's read to connect and when will get the migration and seed data it'll be working well

#### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/182201565

### CODE REVIEW

File: public/api-docs/swagger.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import swaggerJSDoc from 'swagger-jsdoc';
```
Comment (didasmbalanya) [inline public/api-docs/swagger.js]: remove this page from the public folder.

**Review by didasmbalanya — DISMISSED** [review]
This task is for docker, why are there changes on swagger ?

File: public/api-docs/swagger.js (hunk 1 above)
Comment (Julienish14) [reply public/api-docs/swagger.js]: Noted, and done

**Review by Julienish14 — COMMENTED** [review]
done

**Review by ishimweeli — DISMISSED** [review]

**Review by uwishema11 — APPROVED** [review]

**Review by Francoise02 — APPROVED** [review]

**Review by munyanezaarmel — APPROVED** [review]

**Review by Ntare22 — APPROVED** [review]

**Pull request merged**